### DISCUSSION TITLE
What is the best way to add an Image slider / carousel to my Statamic Website?

### DISCUSSION THREAD

**Discussion by uniUbungsAccount** [statamic/cms#7658] [General]
I dont need anything fancy, just different images sliding at the top of my statamic page. Is there a Fieldset/Form for this in Statamic? Or should I do it all in HTML & CSS? Any guides would be much appreciated, thanks!

**Comment by ryanmitchell** [comment]
You could do it with an [Assets](https://statamic.dev/fieldtypes/assets#overview) field type in Statamic's CP and then you output it however you need in the front end template.

**Reply by uniUbungsAccount** [reply]
Thanks a lot for your reply; I think I have a vague idea but I am still quite unsure how I output that to my needs in the front end template. Cant find good documentation for that specific issue either; which makes both sense and my life harder. Sorry, i am a bit of a noob at statamic.

**Reply by jasonvarga** [reply]
On the page Ryan linked to, there is a "templating" section that explains how you loop over the selected assets. In that example it just uses `<img>` tags but you could use `<div>`s or whatever makes sense for your slider.

```antlers
<div class="carousel">
{{ carousel_assets_field }}
  <div class="carousel-image">
    <img src="{{ url }}" />
  </div>
{{ /carousel_assets_field }}
</div>
```
